qemu/tests/cfg/balloon_check.cfg

```diff
@@ -54,6 +54,16 @@
             check_image = yes
         - balloon-stop_continue:
             sub_test_after_balloon = "stop_continue"
+        - iommu_enabled:
+            only q35
+            only balloon_evict_and_enlarge
+            no WinXP WinVista Win7 Win8 Win8.1 Win2000 Win2003
+            no Win2008 Win2008..r2 Win2012 Win2012..r2
```

Comment:
iommu works on modern mode,it should be added in cfg

Reply:
Yes, here specify only q35, q35 default is modern mode.

Reply:
@xiagao  morden mode is the default mode in automation now.

Reply:
understand.